README.md

```diff
@@ -6,6 +6,108 @@
 [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-guidewire.insnow.svg)](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/commits/master)
 [![GitHub Issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-library/module/github.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-library/labels/module%2Fguidewire.insnow)
 
+[Guidewire InsuranceNow](https://www.guidewire.com/products/insurancenow) is a cloud-based insurance platform offering comprehensive tools for policy, billing, and claims management, designed to streamline operations and improve customer service in the insurance industry.
+
+The `ballerinax/guidewire.insnow` package provides a client API to connect to Guidewire InsuranceNow cloud API using Ballerina.
+
+The Ballerina Guidewire InsuranceNow connector is compatible with the Guidewire InsuranceNow `2023.3 (Innsbruck)` REST API version.
+
+## Setup guide
+
+To use the Ballerina Guidewire InsuranceNow connector, access to the Guidewire API is required. Access is granted to partners or customers of the Guidewire platform. To become a partner or customer, reach out to Guidewire's sales team via the [Guidewire website](https://www.guidewire.com/contact-us/). For detailed instructions on obtaining authentication/authorization keys, please go through the [portal development guide](https://docs.guidewire.com/cloud/in/20233/portaldev/PortalDevelopment/topics/c_overview_portal_development.html).
+
+## Quickstart
+
+To use the Guidewire InsuranceNow connector in your Ballerina application, modify the `.bal` file as follows:
+
+### Step 1: Import the connector
+
+Import `ballerinax/guidewire.insnow` module into your Ballerina project.
+
+```ballerina
+import ballerinax/guidewire.insnow;
+```
+
+### Step 2: Instantiate a new connector
+
+Create an `insnow:Client` object with your domain URL and relevant authentication options.
+
+```ballerina
+configurable string username = ?;
+configurable string password = ?;
+
+insnow:ConnectionConfig insnowConfig = {
+    auth: {
+        username,
+        password
+    }
+};
+
+insnow:Client insnow = check new (insnowConfig, "https://<your-domain>/coreapi/v5");
+```
+
+### Step 3: Invoke the connector operations
+
+Now, utilize the available connector operations.
+
+#### List applications
+
+```ballerina
+insnow:ListApplication applications = check insuranceNow->/applications();
```

Comment:
```suggestion
insnow:ListApplication applications = check insNow->/applications();
```

Reply:
You have initialized the client as `insNow` in previous section.

Reply:
Fixed with [39ffe40e0f70f5f3af651fee10e11e3952c9d49c](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/pull/14/commits/39ffe40e0f70f5f3af651fee10e11e3952c9d49c)